bridge/mattermost/mattermost.go

```diff
@@ -23,6 +23,10 @@ type Bmattermost struct {
 	avatarMap map[string]string
 }
 
+const (
+	MATTERMOST_PLUGIN = "mattermost.plugin"
```

Comment:
Please use standard camel-case.

Reply:
Shouldn't it be exportable like config.EVENT_BLAH? Sorry, these conventions are new to me :)

Reply:
Ok cool, just so i understand and am learning consistently, then the constants in `config.EVENT_*` are the anti-pattern that we should fix later?

Reply:
Yes indeed.